src/System.Buffers/ref/System.Buffers.csproj

```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
   	<!-- Must match version supported by frameworks which support 4.0.* inbox.
          Can be removed when API is added and this assembly is versioned to 4.1.* -->
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
```

Comment:
We should bump minor version of the netcoreapp assembly since it has more API than last version (as well as netstandard version)